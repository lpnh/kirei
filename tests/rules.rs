mod helper;

// === Whitespace

#[test]
fn html_trailing_space() {
    format_and_snapshot!("rules/whitespace/html_trailing_space.html");
}

#[test]
fn html_blank_line() {
    format_and_snapshot!("rules/whitespace/html_blank_line.html");
}

#[test]
fn tmpl_trailing_space() {
    format_and_snapshot!("rules/whitespace/tmpl_trailing_space.html");
}

#[test]
fn entity() {
    format_and_snapshot!("rules/whitespace/entity.html");
}
