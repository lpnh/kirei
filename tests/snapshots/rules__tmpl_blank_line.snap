---
source: tests/rules.rs
description: "\n    {%  \n            if \n\n\n        foo  \n                ~%}   \n\n    This    is\n\n\n    a    multi-line\n\n            {{-    \n\n        bar \n    -}}   \n\n    {%   \n\n        else if     another   \n\n            -%}         \n\n\n            if block  surrounded    \n\n        by whitespaces \n\nand \n\n                    blank lines !   \n\n      {%- \n             endif  \n\n            %}\n\n"
---
{% if foo ~%}
    This is a multi-line {{- bar -}}
{% else if another -%}
    if block surrounded by whitespaces and blank lines !
{%- endif %}
